assets/component-mega-menu.css

```diff
@@ -55,10 +54,11 @@
 
 .mega-menu__link--level-2 {
   font-size: 1.4rem;
+  font-weight: bold;
 }
 
 .mega-menu__link--level-2:not(:only-child) {
-  margin-bottom: 0.8rem;
+  margin-bottom: 0rem;
 }
```

Comment:
This whole thing can be removed from what I'm seeing. We were applying a `margin-bottom` before but if we don't need one, then we can remove the whole declaration. 

When I remove it, it looks the same: 
![](https://screenshot.click/30-27-srpfq-4wqsx.gif)

Reply:
Valid point! I went ahead and removed the line of code :)